### PR TITLE
feat: update default environment variable names

### PR DESCRIPTION
**Pull request by veryspry**
### Summary

The Gatsby Shopify cloud integration uses a different set of environment variables. This unifies the variables names for a more understandable, more intuitive experience when using the cloud integration.

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1,5 +1,5 @@
 GATSBY_DEMO_STORE=false
-GATSBY_SHOPIFY_STORE_URL=your-url.myshopify.com
```
Comment (ascorbic) [inline .env.example:2]: This needs to have the `GATSBY_` prefix, because we need runtime access

Comment (ascorbic) [reply .env.example:2]: If it's critical that is has this name, then we need to assign it to another env var at build time.

Comment (veryspry) [reply .env.example:2]: Got it. Using the `GATSBY_` prefix shouldn't be a problem. I'll make the change!

File: .env.example [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
 GATSBY_STOREFRONT_ACCESS_TOKEN=XXX
-SHOPIFY_ADMIN_API_KEY=XXX
-SHOPIFY_ADMIN_PASSWORD=XXX
\ No newline at end of file
+GATSBY_SHOPIFY_STORE_URL=your-url.myshopify.com
+SHOPIFY_API_KEY=XXX
+SHOPIFY_SHOP_PASSWORD=XXX
```
Comment (ascorbic) [inline .env.example:5]:
```suggestion
SHOPIFY_SHOP_PASSWORD=shppa_xxxxx
```

**Pull request merged**